### ISSUE TITLE
AVT 2 - Dropdown fails WCAG 2.1 for color contrast. 

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4374]
Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React

Detailed Description
Go to the React Dropdown Component
Start CCA and compare the highlighted option text against the menu and it fails 1.4.11.
<img width="918" alt="Screen Shot 2019-10-17 at 1 56 58 PM" src="https://user-images.githubusercontent.com/21676914/67041145-a43d3900-f0ea-11e9-810e-2a173e2f9d34.png">
Also, the Menu contrast against the page background also fails.
<img width="912" alt="Screen Shot 2019-10-17 at 1 57 30 PM" src="https://user-images.githubusercontent.com/21676914/67041202-bdde8080-f0ea-11e9-8d2f-92fef4c13d26.png">

**Comment by dakahn** [comment]
similar #4376

**Comment by joshblack** [comment]
Looking at this now, it seems like the selected item issue report is checking the wrong colors. I believe it would be:

Foreground: #161616
Background: #f4f4f4

![image](https://user-images.githubusercontent.com/3901764/69459108-a1abb000-0d36-11ea-99af-2964d91f4a59.png)

**Comment by aagonzales** [comment]
Yup, agree with @joshblack. Appears the wrong colors are being sampled. And same with this issue https://github.com/carbon-design-system/carbon/issues/4376

**Comment by aagonzales** [comment]
@snidersd how are you sampling these colors? Are you just using the eye dropper? It might be better if you inspected the code and got the hex codes that way instead. We check all our text colors of contrast and have built it into our tokens. `text-03` is the only text token that won't pass accessibility and that's because its for placeholder text.

**Comment by aagonzales** [comment]
I'm going to close this because the text is correct contrast ratio and we have a separate issue to research and address the 3:1 non-text contrast issues from the system level.
